plugins/inputs/aerospike/aerospike_test.go

```diff
@@ -48,3 +48,19 @@ func TestAerospikeStatisticsPartialErr(t *testing.T) {
 	assert.True(t, acc.HasMeasurement("aerospike_namespace"))
 	assert.True(t, acc.HasIntField("aerospike_node", "batch_error"))
 }
+
+func TestAerospikeParseValue(t *testing.T) {
+	// uint64 with value bigger than int64 max
+	val, err := parseValue("18446744041841121751")
+	assert.NotNil(t, err)
```

Comment:
shouldn't we also assert that `val` _is_ nil?

Also in the case of errors you should use assert.NoError for more informative failures